samples/spm/README.rst

```diff
@@ -8,10 +8,10 @@ Secure Partition Manager
    :depth: 2
 
 The Secure Partition Manager sample provides a reference use of the System Protection Unit peripheral.
-This firmware is required to set up an nRF device with Trusted Execution (|trusted_execution|) so that it can run user applications in the non-secure domain.
+This firmware sets up an nRF device with Trusted Execution (|trusted_execution|) so that it can run user applications in the non-secure domain.
```

Comment:
Using non-secure domain and secure domain can be problematic for future documentation efforts, where secure domain would mean something else.

We should use the ARM terminology here for "non-secure processing environment (NSPE)" and "secure processing environment SPE".

Since that is already in use by other parts of the documentation I suggest we create a follow-up task to update this.

Reply:
https://projecttools.nordicsemi.no/jira/browse/NCSDK-15298